apps/dalmatiner_frontend/src/dalmatiner_idx_handler.erl

```diff
@@ -29,7 +29,9 @@ handle(Req, State) ->
                     lager:warning("Error in query [~s]: ~p", [Q, E]),
                     StatusCode = error_code(E),
                     {ok, Req2} =
-                        cowboy_req:reply(StatusCode, [], Error, Req1),
+                        cowboy_req:reply(StatusCode,
+                                        [{<<"content-type">>, <<"text/plain">>}],
```

Comment:
Line 33 is too long:                                         [{<<"content-type">>, <<"text/plain">>}],.